src/query-tests.ts

```diff
@@ -200,13 +218,103 @@ const queryTests: Array<QueryTest> = [
-        course_topic_question.course_topic_question_active = true;
+        course_topic_question_webwork_question_ww_path NOT LIKE 'Library/%' AND
+        course_topic_question_webwork_question_ww_path NOT LIKE 'Contrib/%' AND
+        course_topic_question_webwork_question_ww_path NOT LIKE 'webwork-open-problem-library/%' AND
+        course_topic_question_webwork_question_ww_path NOT LIKE 'private/our/%' AND
+        course_topic_question_webwork_question_ww_path NOT LIKE 'private/my/%' AND
+        course_topic_question_webwork_question_ww_path NOT LIKE 'private/templates/%' AND
+        course_topic_question_webwork_question_ww_path NOT LIKE 'private/rederly/%' AND
+        course_topic_question.course_topic_question_active = true AND
+        course_topic_question_errors IS NULL;
+        `,
+        expectedResults: [{
+            count: '0'
+        }],
+        message: ((test: QueryTest, result: Array<any>): string => {
+            if(result.length !== 1) {
+                return `Expected 1 row but got ${result.length}`;
+            }
+            return `Expected ${test.expectedResults?.[0].count} and received ${result[0].count}`;
+        })
+    },
+    {
+        name: 'Exam problems with incorrect prefix',
+        query: `
+        SELECT COUNT(*) FROM
+        (
+            SELECT
+            course_question_assessment_info_id,
+            UNNEST(course_question_assessment_info_additional_problem_paths) as course_question_assessment_info_additional_problem_paths,
+            course_question_assessment_info_errors,
+            course_question_assessment_info_active
+            FROM course_question_assessment_info
+        ) ss
+        WHERE
+        course_question_assessment_info_additional_problem_paths NOT LIKE 'Library/%' AND
+        course_question_assessment_info_additional_problem_paths NOT LIKE 'Contrib/%' AND
+        course_question_assessment_info_additional_problem_paths NOT LIKE 'webwork-open-problem-library/%' AND
+        course_question_assessment_info_additional_problem_paths NOT LIKE 'private/our/%' AND
+        course_question_assessment_info_additional_problem_paths NOT LIKE 'private/templates/%' AND
+        course_question_assessment_info_additional_problem_paths NOT LIKE 'private/rederly/%' AND
+        course_question_assessment_info_additional_problem_paths NOT LIKE 'private/my/%' AND
+        course_question_assessment_info_active = true AND
+        course_question_assessment_info_errors IS NOT NULL;
+        `,
+        expectedResults: [{
+            count: '0'
+        }],
+        message: ((test: QueryTest, result: Array<any>): string => {
+            if(result.length !== 1) {
+                return `Expected 1 row but got ${result.length}`;
+            }
+            return `Expected ${test.expectedResults?.[0].count} and received ${result[0].count}`;
+        })
+    },
+    {
+        name: 'Missing last influencing attempt (based on num attempts)',
```

Comment:
Should we do missing based on joined workbooks before end date, or is that too annoying?

Reply:
That was one of the tests I wanted to do but it ended up being really annoying, I do want to do it eventually